library/silverpeas

```diff
@@ -0,0 +1,9 @@
+Maintainers: Miguel Moquillon <miguel.moquillon@silverpeas.org> (@mmoqui)
+GitRepo: https://github.com/Silverpeas/docker-silverpeas-prod.git
+
+Tags: 6.0-alpha2, latest
+GitCommit: 35af9d88f66d84e4e26ebde6f59cabedfc66f387
+
+Tags: 6.0-alpha1
+GitCommit: b87b3bb7e28e61cd9d76262e897451e2b466d265
```

Comment:
This version doesn't have the same improvements that the `6.0-alpha2` Dockerfile has.  Is this version necessary?  The versions in this file control the "Supported Versions" list that gets generated into the docs that show up on the Docker Hub.

We usually only recommend keeping the versions that are "supported" by upstream; ie which versions will get a minor or patch bump simultaneously.  Some projects only keep one version, like [wordpress](https://github.com/docker-library/wordpress/)*, while others like [pypy](https://github.com/docker-library/pypy/) have 2 versions, and then images like [python](https://github.com/docker-library/python/) actively update 5 versions.

<sub>* one version of WordPress is maintained with multiple variations under different Linux distributions, versions of php, and server types (fpm/apache)</sub>